src/features/components/templates/navigation/main-nav/content/resources/data.ts

```diff
@@ -38,6 +38,13 @@ export const resourcesNavSectionContent: SmartNavContent[] = [
     },
     {
         id: 5,
+        data: {
+            title: '_t_Acuity_t_',
+            url: { type: 'internal', to: '/acuity/' },
+        },
+    },
+    {
+        id: 6,
         data: {
             title: '_t_Status page_t_',
             url: { type: 'non-company', href: 'https://deriv.statuspage.io', show_eu_modal: true },
```

Comment:
can these url be imported from contants?

Reply:
@habib-deriv  yes can since its not my change do I need to move this to constants?

Reply:
Yes please

Reply:
sure i will do @habib-deriv